epm/Distribution/Client/Setup.hs

```diff
@@ -1346,6 +1348,11 @@ installOptions showOrParseArgs =
            installSymlinkBinDir (\v flags -> flags { installSymlinkBinDir = v })
            (reqArgFlag "DIR")
 
+      , option [] ["eta-patches-directory"]
```

Comment:
Because this is for `epm` and we know that it's for Eta, maybe we should call this `patches-directory` instead? It'll save some typing as well.